javascript/javascript1/week2/studentmanager.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+const queen='Margrethe II';
+function addStudentToClass(studentName) {
+  if (!class07Students.includes(queen)){
+      class07Students.push(queen);
+      console.log(`queen ${queen} has been added to the the class07`);
+} if  (class07Students.length>6){
+    console.log("Cannot add more students to class 07");
+      } else if (class07Students.length){
+      console.log(`student ${studentName} is included in the class07`)
+  }if (studentName===''){
+    console.log("you cannot add an empty string to the class")
+}else{
+      console.log(class07Students.push(studentName))
```

Comment:
You are saying here, that if the student's name is not an empty string, push the student's name to the array. And you are not checking if there are more than 6 in the array already.